### PR TITLE
[stable/spinnaker] Fix chart not being upgradable

### LINKED ISSUE #13977: [stable/spinnaker] can't upgrade spinnaker
**Describe the bug**

The StatefulSet spinnaker-halyard contains a mutable value in
spec.volumeClaimTemplates.metadata.labels (i.e, (chart name)-(version))
so that we cannot upgrade the spinnaker chart.

This is the same issue as https://github.com/helm/charts/issues/7803

**Version of Helm and Kubernetes**:

```
❯ helm version
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
```
```
❯ kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-19T22:12:47Z", GoVersion:"go1.12.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.6-eks-d69f1b", GitCommit:"d69f1bf3669bf00b7f4a758e978e0e7a1e3a68f7", GitTreeState:"clean", BuildDate:"2019-02-28T20:26:10Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:

stable/spinnaker

**What happened**:

Could not upgrade spinnaker from 1.8.1 to 1.8.2 with the following error:

```
StatefulSet.apps "spinnaker-spinnaker-halyard" is invalid: spec: Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden
```

**What you expected to happen**:

Be able to upgrade spinnaker from 1.8.1 to 1.8.2

**How to reproduce it** (as minimally and precisely as possible):

Install spinnaker 1.8.1 and upgrade it.

**Anything else we need to know**:

See https://github.com/helm/charts/issues/7803

### PR DESCRIPTION
**Pull request by skaji**
#### What this PR does / why we need it:

Fix chart not being upgradable;
don't include mutable values in spec.volumeClaimTemplates.metadata.labels

#### Which issue this PR fixes

- fixes #13977

#### Special notes for your reviewer:

See https://github.com/helm/charts/issues/7803 for explanation.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/spinnaker/templates/statefulsets/halyard.yaml [hunk 1]
```diff
@@ -143,7 +143,9 @@ spec:
   - metadata:
       name: halyard-home
       labels:
-{{ include "spinnaker.standard-labels" . | indent 8 }}
+        app: {{ include "spinnaker.fullname" . | quote }}
```
Comment (dwardu89) [inline stable/spinnaker/templates/statefulsets/halyard.yaml]: Would it be better to place these values in a helper, then reference it to be something as follows?
`{{ include "spinnaker.standard-labels-base" . | indent 8 }}`

I know it is another layer of abstraction, but we can use the base labels for other parts of the chart where necessary.

Comment (skaji) [reply stable/spinnaker/templates/statefulsets/halyard.yaml]: @dwardu89 Thanks for your review.
The commit 50efbac defines "spinnaker.standard-labels-base", and use it.

**Comment by skaji** [comment]
@dwardu89 I've merged master branch, and bumped version again.
Could you review this again please?

**Pull request merged**